Split/Network/Streaming/SyncManager.swift

```diff
@@ -63,20 +67,28 @@ class DefaultSyncManager: SyncManager {
         switch pushEvent {
         case .pushSubsystemUp:
             Logger.d("Push Subsystem Up event message received.")
+            reconnectStreamingTimer.cancel()
             synchronizer.syncAll()
             synchronizer.stopPeriodicFetching()
             isPollingEnabled.set(false)
             Logger.i("Polling disabled")
 
         case .pushSubsystemDown:
             Logger.d("Push Subsystem Down event message received.")
+            reconnectStreamingTimer.cancel()
             enablePolling()
+            pushNotificationManager.stop()
```

Comment:
Recheck, but a `pushSubsystemDown` event emitted due to a `streamingPaused` OCCUPANCY notification, should not disconnect SSE, i.e., I think we shouldn't call `pushNotificationManager.stop()` here.

Reply:
Yup doc, same to previous comment. Thanks!